src/view.c

```diff
@@ -627,7 +638,12 @@ struct view_s* create_view(const char* name, const long pos[DIMS], const long di
 	v->pos = xmalloc(DIMS * sizeof(long));
 
 	for (int i = 0; i < DIMS; i++)
-		v->pos[i] = (NULL != pos) ? pos[i] : 0;
+			v->pos[i] = (NULL != pos) ? pos[i] : 0;
+
+	if ( iopts.r > 0) {
```

Comment:
here is a space too much and in the next lines there are spaces missing around the operators